### PR TITLE
feat: Windows.Devices.Radios common structs/enums

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): #3176, part #3045

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
Windows.Devices.Radios is not implemented.

## What is the new behavior?
Structs/enums for Windows.Devices.Radios are implemented.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information
First part of Windows.Devices.Radios.Radio.GetRadiosAsync() implementation.
Enums defined for all platforms.

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UWP/Devices/Radios/RadioState.cs
Comment (MartinZikmund) [inline src/Uno.UWP/Devices/Radios/RadioState.cs:9]: Remove the `#if` here

**Review by MartinZikmund — CHANGES_REQUESTED** [review]

File: src/Uno.UWP/Devices/Radios/RadioState.cs
Comment (pkar70) [reply src/Uno.UWP/Devices/Radios/RadioState.cs:9]: Of course. Sorry.

**Review by MartinZikmund — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UWP/Devices/Radios/Radio.cs
Comment (jeromelaban) [inline src/Uno.UWP/Devices/Radios/Radio.cs:7]: The Radio class is not implemented, so there's no point removing the not implemented attributes.

Comment (pkar70) [reply src/Uno.UWP/Devices/Radios/Radio.cs:7]: This structs are common for all platforms, I extracted it to this PR.
I have ready implementation of GetRadiosAsync(), for Android only, and I want it to be in another PR.

Comment (jeromelaban) [reply src/Uno.UWP/Devices/Radios/Radio.cs:7]: That's fine, then you can add this class to this PR.

Comment (pkar70) [reply src/Uno.UWP/Devices/Radios/Radio.cs:7]: Android GetRadiosAsync() is in #5596.
Why mix all platforms PRs with single platforms PRs?

All my single platform PRs are still unmerged, for over year now... But "all platforms" structs from these PRs, extracted to simpler PRs, are being merged. Maybe as help for others, who would write same PRs as I, but multiplatform.

Comment (jeromelaban) [reply src/Uno.UWP/Devices/Radios/Radio.cs:7]: Please remove this class from this PR, and add it to the other PR. This class is not implemented until the other PR is merged.

Comment (pkar70) [reply src/Uno.UWP/Devices/Radios/Radio.cs:7]: As #5596 included all changes from this PR (I didn't intend to, but git/vstudio make it so) , maybe skip this PR and deal only with #5596?

**Comment by pkar70** [comment]
As this PR has two approvals, can it be marked as "ready to merge" and merged?

**Comment by pkar70** [comment]
This PR has two approvals, no build errors, so why it is not merged?

**Comment by jeromelaban** [comment]
You've marked the comment as resolved and it's not resolved. The class you've added is not implemented, we cannot merge this PR. The other PR may contain the associated code, but have not reviewed that one.

File: src/Uno.UWP/Devices/Radios/Radio.cs
Comment (pkar70) [reply src/Uno.UWP/Devices/Radios/Radio.cs:7]: All properties of class Radio are implemented here (#5589), code is not implemented as code is platform related (and properties are same for all platforms).
But, when I restrict this PR to enums only, would it be accepted/merged?

**Comment by pkar70** [comment]
Radio.cs properties definition removed, so only enums left.

**Comment by pkar70** [comment]
For merge, all checks should be successful? So, please restart build - as problems are not related to this PR.

**Comment by pkar70** [comment]
please restart build - problems are not related to this PR.

**Comment by pkar70** [comment]
Seems like `Retrying 'FindPackagesByIdAsync' for source 'https://pkgs.dev.azure.com/azure-public/3ccf6661-f8ce-4e8a-bb2e-eff943ddd3c7/_packaging/be90bfa8-5c00-452b-ba9f-226ce4366992/nuget/v3/flat2/uno.uitest.selenium/index.json'.
  An error occurred while sending the request.
    Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host..
    An existing connection was forcibly closed by the remote host.` is not related to PR....

**Pull request merged**
